### DISCUSSION TITLE
publish new release more quickly at https://netbeans.apache.org/download/

### DISCUSSION THREAD

**Discussion by wumpz** [apache/netbeans#5554] [General]
### Description

Netbeans 17 was released two days ago. Now many download sites I found (e.g. https://www.techspot.com/downloads/7212-netbeans.html, https://www.codelerity.com/netbeans/, ..) already deliver Netbeans 17. But the one and only homepage https://netbeans.apache.org/download/ is still old, at Netbeans 16.

**Comment by neilcsmith-net** [comment]
Moved this to discussions, but don't disagree.

The official release is on ASF infrastructure at - https://dlcdn.apache.org/netbeans/netbeans/17/ and https://dlcdn.apache.org/netbeans/netbeans-installers/17/

That will always be the first place they will appear, and you also have signatures and checksums to verify the files.  I would always use ASF infrastructure rather than a third-party for the ASF releases.

The Codelerity installers are different - they are packaged with a JDK runtime and built in a different way.  I also have the benefit of handling both sides, so generally they're built in advance and made available as soon as possible after the release is moved into place.

Yes, we need to get better at having the bulk of the NetBeans website ready to go in advance of the vote closing!

**Comment by tofflos** [comment]
I also agree on this one. I'd like to provide feedback on release candidates but compiling from source is too much work. Also I think feedback on build, packaging and distribution is valuable during a release candidate phase. Could posting builds as assets in the Github release be an option? I think I'd prefer that over downloading them from the website.

**Reply by neilcsmith-net** [reply]
@tofflos release candidates are linked at #6581
